__tests__/app.test.js

```diff
@@ -200,7 +200,58 @@ describe("/api/articles/:article_id", () => {
   });
 });
 
-describe("/api/users", () => {
+describe("/api/articles", () => {
+  describe("GET", () => {
+    test("status: 200 - responds with an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(Array.isArray(articles)).toBe(true);
+          expect(articles).toHaveLength(12);
+        });
+    });
+    test("status: 200 - article objects in response have these properties: author, title, article_id, topic, created_at, votes", () => {
```

Comment:
If you wanted to clean up a bit this test could be combined with the previous test.

Asserting against the length then having your assertions within the forEach as well :)